loads/src/main/java/org/mousephenotype/cda/loads/create/extract/cdabase/steps/AlleleProcessorAbstract.java

```diff
@@ -163,12 +163,12 @@ protected Allele setBiotypeSkipAlleleIfNoBiotypeFound(Allele allele) {
      *
      * @return the allele being processed, with the biotype component set.
      *
-     * @throws DataImportException
+     * @throws DataLoadException
      */
-    protected Allele setBiotypeMouseMutants(Allele allele) throws DataImportException {
+    protected Allele setBiotypeMouseMutants(Allele allele) throws DataLoadException {
```

Comment:
There are alleles that are not tm1a or tm1e.  There could be tm1 or tm1.1 or tm2a ... or others. We will need to account for those as well